src/components/alert/alert.scss

```diff
@@ -117,7 +117,6 @@ ion-alert input {
 
   font-size: inherit;
   line-height: initial;
-  text-align: left;
```

Comment:
https://developer.mozilla.org/es/docs/Web/CSS/text-align

start is a experimental value, left is needed as fallback value